modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -30,7 +30,7 @@ You can deploy {lvms} using one of the following:
 [id="lvms-deployment-requirements-for-sno-ran_{context}"]
 == Requirements
 
-Before deploying {lvms}, ensure that every managed cluster has dedicated disks that are used to provision storage.
+Before deploying {lvms}, ensure that every managed cluster has dedicated disks that are used to provision storage. {lvms} uses only those disks that are empty and do not contain file system signatures.
```

Comment:
I think this almost hits it but not quite. It will attempt to use all disks and fail if they are containing any signatures except when the "wipe" option is given.

Reply:
Does that mean that when wipe option is used, even the file signatures will be wiped off and thereby making the disks free of any file signatures.

If this is the case, then the statement will still hold true. 
How about adding the following line.
"To ensure that the disks are empty and do not contain file system signatures, wipe the disks before using them. "

Reply:
That would work yes. You can also add that you can alternatively use that flag (if you want to provide a reference). Your call!

Reply:
Talking about the flag in this sec seems like an information overload .

I have added the same info in the desc for the `forceWipeDevicesAndDestroyAllData` field. This info provides more context.